src/resources/Catalogs/CatalogInterfaces.ts

```diff
@@ -36,6 +36,8 @@ export interface CatalogConfigurationModel {
 export interface IAssociatedCatalogModel {
     id: string;
     name: string;
+    sourceId?: string;
```

Comment:
`sourceId` shouldn't be optional, right? :thinking:

Reply:
It is now in the model since we haven't ditched the old `sourceIds` scope. We should change it to required when we do in the catalog model as well